charmcraft/templates/init-fastapi-framework/tox.ini.j2

```diff
@@ -0,0 +1,84 @@
+# Copyright {{ year }} {{ author }}
+# See LICENSE file for licensing details.
```

Comment:
Shouldn't be worth to move this `tox` configuration to the `pyproject.toml` as you require `4.0.0` minimal version anyway? 

https://tox.wiki/en/latest/config.html#pyproject-toml-native

Reply:
Possibly, but out of scope as this is just copying the tox files from the other templates.